app/src/main/java/com/futurice/freesound/feature/home/HomeFragmentViewModel.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.feature.home;
+
+import com.futurice.freesound.feature.common.Navigator;
+import com.futurice.freesound.network.api.model.UserResult;
+import com.futurice.freesound.viewmodel.BaseViewModel;
+
+import android.support.annotation.NonNull;
+
+import io.reactivex.Single;
+import io.reactivex.disposables.CompositeDisposable;
+
+import static com.futurice.freesound.utils.Preconditions.get;
+
+class HomeFragmentViewModel extends BaseViewModel {
+
+    @NonNull
+    private final Navigator navigator;
+
+    @NonNull
+    private final Single<UserResult> homeUser;
+
+    HomeFragmentViewModel(@NonNull final UserDataModel userDataModel,
+                          @NonNull final Navigator navigator) {
+        this.navigator = get(navigator);
+        homeUser = get(userDataModel).getHomeUser()
+                                     .cache();
```

Comment:
I would like to hear more about the `cache` operator - perhaps an upcoming AW topic?